packages/ui/src/mainbar.tsx

```diff
@@ -0,0 +1,82 @@
+'use client';
+
+import { useState } from 'react';
+import MenuIcon from '@mui/icons-material/Menu';
+import {
+	AppBar,
+	Drawer,
+	IconButton,
+	InputBase,
+	Toolbar,
+	Typography,
+} from '@mui/material';
+import Link from 'next/link';
+
+import Sidebar from './sidebar';
+import SigninWithGoogleButton from './signInWithGoogleButton';
+
+const Navbar = () => {
+	const [sidebarOpen, setSidebarOpen] = useState(false);
+
+	const handleDrawerToggle = () => {
+		setSidebarOpen(!sidebarOpen);
+	};
+
+	return (
+		<>
+			<AppBar position="static" component="nav" className="appbar">
+				<Toolbar>
+					<IconButton
+						color="inherit"
+						aria-label="open drawer"
+						edge="start"
+						onClick={handleDrawerToggle}
+					>
+						<MenuIcon />
+					</IconButton>
+					<Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
+						<Link
+							href="/"
+							style={{ textDecoration: 'none', color: 'white', marginLeft: 10 }}
```

Comment:
is there anyway to avoid using style this way? e.g `styled-component`, or @material's way like `sx={{ ... }}`?